worldengine/world.py

```diff
@@ -416,66 +421,67 @@ def tiles_around_many(self, pos_list, radius=1, predicate=None):
     #
 
     def start_mountain_th(self):
-        return self.elevation['thresholds'][2][1]
+        return self.elevation['thresholds'][3][1]
 
-    def is_mountain(self, pos):
+    def is_mountain(self, pos):  # general test
         if self.ocean[pos[1], pos[0]]:
             return False
-        if len(self.elevation['thresholds']) == 4:
-            mi = 2
-        else:
-            mi = 1
-        mountain_level = self.elevation['thresholds'][mi][1]
+        mountain_level = self.elevation['thresholds'][2][1]  # hills are mountains, too
         x, y = pos
-        return self.elevation['data'][y][x] > mountain_level
+        return mountain_level <= self.elevation['data'][y, x]
 
-    def is_low_mountain(self, pos):
-        if not self.is_mountain(pos):
+    def is_hill(self, pos, atleast=False):
```

Comment:
I chose to add an `atleast` parameter instead of adding another four functions that basically contain copied code.